### DISCUSSION TITLE
Front End js drop down menu

### DISCUSSION THREAD

**Discussion by evertonProgramadorCriativo** [frontendbr/forum#1608] [Perguntas e Respostas]
ola amigos eu estou querendo fazer uma exceção para que quando eu clica no input  o menu suspenso  nao sair   e deixa o usuario escreve  dentro do input 
![Screenshot_15](https://user-images.githubusercontent.com/55323538/68860121-9ec21700-06c7-11ea-9b47-51c91eb0b513.png)

-------------js--------script---------------------------
  function myFunction2() {
    document.getElementById("int").classList.toggle("show");
  }
  
  window.onclick = function(event) {
    if (!event.target.matches('.dropbtn')) {
      var dropdowns = document.getElementsByClassName("dropdown-content");
      var i;
      for (i = 0; i < dropdowns.length; i++) {
        var openDropdown = dropdowns[i];
        if (openDropdown.classList.contains('show')) {
         
         
         openDropdown.classList.remove('show' );

          
        }
      }
    }
  }

--------------------------------------html----------------

<div class="dropdown nenos-u-margin-left-samll">
                <button onclick="myFunction2()" class="dropbtn">
                    <i class="far fa-question-circle"></i>
                </button>
                <div id="int" class="dropdown-content">
                    <i class="fas fa-sort-down icon-rotate icon-rotate--samll"></i>

                    <div class="dropdown__option-menu">
                        <a href="#" class="dropdown__option-menu--btnLeft">Ajuda Rapida</a>
                        <a href="#" class="dropdown__option-menu--btnRight">Central de ajuda</a>
                    </div>

                    <form action="#" id="formm" class="form__dropdownSmal">
                        <input type="text" class="form__dropdownSmal--search" placeholder="Como podemos ajudar ?"
                            required>
                        <button class="form__dropdownSmal--btnSearch">
                            <i class="fas fa-search"></i>
                        </button>
                    </form>

                    <a href="#" class="dropdown__btn">AJUDA COM ESSA PAGINA</a>
                    <a href="#" class="dropdown__btn">Linha do Tempo</a>
                    <a href="#" class="dropdown__btn">Bate-papo e mensagens</a>
                    <a href="#" class="dropdown__btn">Segurança da conta</a>
                    <a href="#" class="dropdown__btn">Privacidade</a>
                    <div class="dropdown__btnMain">
                        <a href="#" class="dropdown__btnMain--Text">
                            <i class="fas fa-cog"></i>
                            Atalhos de privacidade
                        </a>
                    </div>
                    <div class="dropdown__btnMain">
                        <a href="#" class="dropdown__btnMain--Text">
                            <i class="fas fa-envelope"></i>
                            Caixa dde Entrada do Suporte
                        </a>
                    </div>
                    <div class="dropdown__btnMain">
                        <a href="#" class="dropdown__btnMain--Text">
                            <i class="fas fa-exclamation-circle"></i>
                            Relatar um problema
                        </a>
                    </div>
                </div>
            </div>

-----------------------------------scss-----------------------------------------------
.show {display: block;}

**Comment by kvnol** [comment]
@evertonProgramadorCriativo beleza? Mano, você pode colocar esse seu código no Codepen? Para conseguirmos visualizar melhor?
